tests/UI/pages/commonPage.js

```diff
@@ -312,10 +312,15 @@ module.exports = class CommonPage {
    * @param isFloat
    * @return {Promise<*>}
    */
-  async sortArray(arrayToSort, isFloat = false) {
+  async sortArray(arrayToSort, isFloat = false, isDate = false) {
```

Comment:
We're adding more and more "special cases", maybe that would be a good idea to refactor this by accepting an object instead of a few arguments ?
WDYT ? (not now in this PR obviously, but as an improvement)

Reply:
Yes i thought about something like below and doing it in this PR but it include updates in different scenarios.
So I will add it to improvement list for UI tests

```js
  async sortArray(arrayToSort, type = 'string') {
    switch(type): {
       case 'float':
         return arrayToSort.sort((a, b) => a - b);

      case 'date':
         return arrayToSort.sort((a, b) => new Date(b) - new Date(a));

      case 'string':
      default:
          return arrayToSort.sort((a, b) => a.localeCompare(b));
  }
```